flake.nix

```diff
@@ -127,6 +127,10 @@
           checks = self'.checks;
           inputsFrom = [ typst ];
 
+          buildInputs = with pkgs; [
+            rust-analyzer
+          ];
```

Comment:
We should probably not use the version from nixpkgs, but rather the one from `rust-toolchain` given `inputsFrom` above.

Reply:
That's interesting. The `craneLib` variable we create above does expose individual components such as `rustc`, `clippy` etc as derivations, but seemingly not `rust-analyzer`. Does it even depend on the Rust version used then? And if not, then are you sure this change would improve your case?